### PR TITLE
feat(inputcheckbox): add optional caption

### PR DESCRIPTION
**Pull request by bethbertozzi**
## Summary

**Input Checkbox**
Shortcut ticket: [sh-204074](https://app.shortcut.com/sci-design-system/story/204074/add-caption-to-checkbox-component)
Add functionality and update like preview linked below. Caption should be optional.

## Checklist

- [ ] Default Story in Storybook
- [ ] LivePreview Story in Storybook
- [ ] Test Story in Storybook
- [ ] Tests written
- [ ] Variables from `defaultTheme.ts` used wherever possible
- [ ] If updating an existing component, depreciate flag has been used where necessary
- [ ] Chromatic build verified by @chanzuckerberg/sds-design

### CODE REVIEW

**Review by clarsen-czi — CHANGES_REQUESTED** [review]
Looking good @bethbertozzi ! The only change is there should be the option to add a caption

**Comment by bethbertozzi** [comment]
@clarsen-czi I coded it so if the caption prop is empty, it is removed 👍

File: src/core/InputCheckbox/index.tsx [hunk 1]
```diff
@@ -109,7 +112,9 @@ const InputCheckbox = (props: CheckboxProps): JSX.Element => {
           {...checkboxProps}
         />
       }
-    />
+    >
+      <div>Conetnes</div>
```
Comment (masoudmanson) [inline src/core/InputCheckbox/index.tsx:116]: I think this is a typo!

**Review by masoudmanson — APPROVED** [review]
Awesome, Thank you @bethbertozzi

**Review by clarsen-czi — APPROVED** [review]
Thanks for clarifying the functionality of the caption @bethbertozzi . This LGTM!!

**Pull request merged**